kuttl-test.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestSuite
 artifactsDir: ./tests/_build/artifacts/
+commands:
+  - command: make undeploy
+  - command: make enable-prometheus-feature-flag deploy
```

Comment:
Could be the test rather placed into `e2e-prometheuscr`. That suite contains the tests for prometheus integration